libclingcon/src/clingcon.cc

```diff
@@ -151,6 +151,36 @@ template<class T>
     }
 }
 
+template<>
```

Comment:
You can just use the iostream version I send you for both int and double parsing. This is getting too long. We can live with the unnecessary allocations during option parsing.